server/README.adoc

```diff
@@ -28,6 +28,28 @@ KROKI_DOT_BIN_PATH:: Path to the `dot` binary (provided by GraphViz)
 KROKI_NOMNOML_BIN_PATH:: Path to the `nomnoml` binary
 KROKI_BLOCKDIAG_HOST:: The host of the blockdiag server
 KROKI_BLOCKDIAG_PORT:: The port of the blockdiag server
+KROKI_SAFE_MODE:: One of:
++
+--
+`SECURE`:: (default) Paranoid sanitization of requests before passing to image server.
```

Comment:
I have used "diagram library" instead of "image server" in the documentation. I'm not a native speaker so maybe "image server" is more appropriated but in this case, we should replace every occurrences of "diagram library" otherwise it might be confusing.

Maybe we can stick with "diagram library" in this pull request and create another one to change the terminology if needed? What do you think?

Reply:
So stick with "diagram library"?

Reply:
```suggestion
`SECURE`:: (default) Paranoid sanitization of requests before passing to the diagram library.
```